packages/e2e-tests/next-app/cypress.json

```diff
@@ -2,5 +2,7 @@
   "baseUrl": "http://localhost:3000",
   "supportFile": "cypress/support/index.ts",
   "responseTimeout": 15000,
-  "requestTimeout": 15000
+  "requestTimeout": 15000,
+  "experimentalNetworkStubbing": true,
```

Comment:
These two options allow using `route2()` for network level interception and also auto-polyfilling fetch